src/components/List/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./index.css";
+import Task from "../Task";
+import TaskType from "../../types/TaskType";
+import PriorityType from "../../types/PriorityType";
+
+interface ListProps {
+  tasks: TaskType[];
+  setTasks: (tasks: TaskType[]) => void;
+  filter: PriorityType;
+}
+const List = ({ tasks, setTasks, filter }: ListProps) => {
+  const deleteTask = (id: string) => {
+    const newTasks = tasks.filter((task) => task.id !== id);
+    setTasks(newTasks);
+  };
+
+  const editTask = (updatedTask: (typeof tasks)[0]) => {
```

Comment:
what does `: (typeof tasks)[0]` mean?

Reply:
i missed it, it was before i created the type